reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+includes a command-line shell, object-oriented scripting language, and a set of tools for executing
+scripts/cmdlets and managing modules. Designed for Cloud, Hybrid-Cloud and on-premises, the latest
+release of PowerShell is packed with performance enhancements and features over previous versions.
+
+Migrate with minimal investment time and lower the risk of failure. PowerShell 7 is designed to work
+fully side-by-side with Windows PowerShell letting you easily and quickly test and compare before
+deployment.
+
+## PowerShell 7 Supported on Windows
+
+PowerShell 7 is ready to deploy for Microsoft Windows client and server operating systems.
+PowerShell 7 is supported on the following Windows operating systems:
+
+- Windows 8.1, and 10
+- Windows Server 2012, 2012 R2, 2016, and 2019
+
+For more information about the supported operating systems and support lifecycle, see the
+[PowerShell Support Lifecycle](/powershell/scripting/powershell-support-lifecycle?view=powershell-7).
+
+## Installing PowerShell 7?
+
+To support the needs and flexibility requirements of IT, DevOps and Development, there are several
+options for installing PowerShell 7. For most administrators working with Windows client or server,
+you can reduce the installation options to the two below:
+
+1. To install PowerShell 7 on your Windows client 8.1/10 computer, open the Microsoft Store and
+   enter `PowerShell 7` in the search bar.
+
+2. To deploy to Windows Server, get the binary **.msi** or **.zip** package. You can download the
+   packages from [GitHub Release page](https://github.com/PowerShell/PowerShell/releases).
+
+> [!NOTE]
+> The **.msi** package is updated and supported with management products such as Microsoft
+> System Center Configuration Manager (SCCM)
+
+For details on installing PowerShell 7 on Windows, macOS or Linux, see [Installing PowerShell](https://aka.ms/Get-PowerShell)
+
+## Running PowerShell 7
+
+PowerShell 7 installs in a new directory on Microsoft Windows to permit side-by-side execution with
+Windows PowerShell 5.1. The new installation directory is added to your path allowing you to execute
+both Windows PowerShell and PowerShell 7. If your migrating from PowerShell Core 6.x to PowerShell
+7, the installation is upgraded and the path replaced.
+
+PowerShell 7 installation path is added to Microsoft Windows:
+
+- Windows PowerShell 5.1 : `C:\Windows\System32\WindowsPowerShell\v1.0` will add
+`%programfiles%\PowerShell\7`
+
+- PowerShell Core 6.x on Windows: `%programfiles%\PowerShell\6` is upgraded to
+ `%programfiles%\PowerShell\7`
+
+In Windows PowerShell, the executable to launch PowerShell is named `powershell.exe`. In version 6
+and above, the executable is changed to support side-by-side execution. The new executable to launch
+PowerShell 7 is `pwsh.exe`. Preview builds will remain in-place as `pwsh-preview` instead of `pwsh`
+under the 7-preview directory.
+
+## Modules and Module paths
```

Comment:
Consider a table highlighting the aspects that make pwsh and powershell side-by-side:

- executable name
- psmodulepaths
- group policy
- event logs

Then go into detail about each one